src/util/bucket_logs_utils.js

```diff
@@ -43,16 +43,17 @@ async function export_logs_to_target(fs_context, s3_connection, bucket_to_owner_
  * This function gets a persistent log file, will go over it's entries one by one,
  * and will upload the entry to the target_bucket using the provided s3 connection
  * in order to know which user to use to upload to each bucket we will need to provide bucket_to_owner_keys_func
- * @param {nb.NativeFSContext} fs_context
+ * @param {Object} global_config
  * @param {AWS.S3} s3_connection
  * @param {string} log_file
  * @param {function} bucket_to_owner_keys_func
  * @returns {Promise<Boolean>}
  */
-async function _upload_to_targets(fs_context, s3_connection, log_file, bucket_to_owner_keys_func) {
+async function _upload_to_targets(global_config, s3_connection, log_file, bucket_to_owner_keys_func) {
```

Comment:
i wouldn't pass a thing called global_config everywhere in the code. this will be a mess and no one can know which global_config this object is.